### DISCUSSION TITLE
Window installation of new version Genesis

### DISCUSSION THREAD

**Discussion by xiyang13** [svenreiche/Genesis-1.3-Version4#104] [Q&A]
Is there a detailed installation manual for window 10?

**Comment by svenreiche** [comment]
Unfortunately it is the same with macos x. At the end you have to compile it in a unix like environment and install the required libraries. These are openmpi, hdf5 with parallel support and the fftw3. None of these come natively for windows nor macos X. However if you can use cmake than it gets a little bit better since it is probing for existing libraries and you can define the  compiler (e.g. on the HPC platform I am running my simulation I have the intel compiler available so it should work also for windows).
